### PR TITLE
MS aged and blind exemption

### LINKED ISSUE #2192: MS aged and blind exemption
$1,500 exemption for each taxpayer who is over age or 65

"n the case of any taxpayer or the spouse of the taxpayer who has attained the age of sixty-five (65) before the close of his taxable year, an additional exemption of One Thousand Five Hundred Dollars ($1,500.00)."

https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=6

Miss. Code Ann. § 27-7-21 (g)
https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=adb5f365-0449-4d49-80f0-22673c0d0165&nodeid=AAPAAFAABAAP&nodepath=%2fROOT%2fAAP%2fAAPAAF%2fAAPAAFAAB%2fAAPAAFAABAAP&level=4&haschildren=&populated=false&title=%C2%A7+27-7-21.+Exemptions+allowed.&config=00JABhZDIzMTViZS04NjcxLTQ1MDItOTllOS03MDg0ZTQxYzU4ZTQKAFBvZENhdGFsb2f8inKxYiqNVSihJeNKRlUp&pddocfullpath=%2fshared%2fdocument%2fstatutes-legislation%2furn%3acontentItem%3a8V3T-78H2-D6RV-H4WK-00008-00&ecomp=8gf5kkk&prid=e1d2bafd-2018-44e8-a4fc-e9e2b08a4daa

### PR DESCRIPTION
**Pull request by SuriLuo**
Fixes #2192

create MS aged and blind exemptions

### CODE REVIEW

**Comment by MaxGhenis** [comment]
Could you run `pip install -U black; make format`? That should fix the extraneous changes

File: changelog_entry.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- bump: minor
+  changes:
+    added:
+    - Mississippi aged and blind exemptions update.
```
Comment (PavelMakarchuk) [inline changelog_entry.yaml:4]:
```suggestion
    - Mississippi aged and blind exemptions.
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+description: MS provides an additional $1.5k exemption for filers or spouse above age 65
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml:1]:
```suggestion
description: Mississippi provides this amount exemption amount for aged filers or spouse.
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+metadata:
+  type: single_amount
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml:3]:
```suggestion
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  label: MS senior exemption age threshold
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml:4]:
```suggestion
  label: Mississippi senior exemption age threshold
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/amount.yaml [hunk 5]
```diff
@@ -0,0 +1,13 @@
+description: Mississippi provides an exemption of this amount per aged filer or spouse.
+metadata:
+  unit: currency-USD
+  label: Mississippi aged exemption
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/amount.yaml:4]:
```suggestion
  label: Mississippi aged exemption amount
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml (hunk 3 above)
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml:3]:
```suggestion
  unit: year
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/blind/amount.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+description: Mississippi provides an exemption of this amount per blind filer or spouse.
+metadata:
+  unit: currency-USD
+  label: Mississippi blind exemption
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/ms/tax/income/exemptions/blind/amount.yaml:4]:
```suggestion
  label: Mississippi blind exemption amount
```

File: policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_aged_exemption.py [hunk 7]
```diff
@@ -0,0 +1,33 @@
+from policyengine_us.model_api import *
+
+
+class ms_aged_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Mississippi aged and blind exemption"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_aged_exemption.py:7]:
```suggestion
    label = "Mississippi aged exemption"
```

File: policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_blind_exemption.py [hunk 8]
```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class ms_blind_exemption(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Mississippi aged and blind exemption"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/ms/tax/income/exemptions/ms_blind_exemption.py:7]:
```suggestion
    label = "Mississippi blind exemption"
```

File: policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+  reference: 
+    - title: Miss. Code Ann. § 27-7-21 (f)
+      href: https://advance.lexis.com/documentpage/?pdmfid=1000516&crid=e1d2bafd-2018-44e8-a4fc-e9e2b08a4daa&nodeid=AAPAAFAABAAP&nodepath=%2FROOT%2FAAP%2FAAPAAF%2FAAPAAFAAB%2FAAPAAFAABAAP&level=4&haschildren=&populated=false&title=%C2%A7+27-7-21.+Exemptions+allowed.&config=00JABhZDIzMTViZS04NjcxLTQ1MDItOTllOS03MDg0ZTQxYzU4ZTQKAFBvZENhdGFsb2f8inKxYiqNVSihJeNKRlUp&pddocfullpath=%2Fshared%2Fdocument%2Fstatutes-legislation%2Furn%3AcontentItem%3A8V3T-78H2-D6RV-H4WK-00008-00&ecomp=8gf5kkk&prid=1ae106b3-6378-4ac5-8291-6c5f714f357c
+    - title: Instructions for Mississippi State FORMS 80-105 AND 80-205
+      href: https://www.dor.ms.gov/sites/default/files/Forms/Individual/80100221.pdf#page=6
+values:
+  2020-01-01: 65
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ms/tax/income/exemptions/aged/age_threshold.yaml:11]: change to 2019 in other yaml files

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**